pkg/minikube/extract/extract.go

```diff
@@ -78,6 +78,16 @@ func newExtractor(functionsToCheck []string) *currentState {
 
 // TranslatableStrings finds all strings to that need to be translated in paths and prints them out to all json files in output
 func TranslatableStrings(paths []string, functions []string, output string) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		exit.WithError("Getting current working directory", err)
```

Comment:
Can this just return an err? It will be difficult to call this in tests otherwise.